http-netty/src/main/java/io/micronaut/http/netty/channel/DefaultEventLoopGroupRegistry.java

```diff
@@ -79,21 +87,23 @@ protected EventLoopGroup eventLoopGroup(EventLoopGroupConfiguration configuratio
                                       configuration.getIoRatio().orElse(null)
                               )).orElseThrow(() -> new ConfigurationException("No executor service configured for name: " + executor));
         } else {
+            ThreadFactory threadFactory = beanLocator.getBean(ThreadFactory.class, Qualifiers.byName(configuration.getName()));
```

Comment:
This can be replaced by `@Parameter ThreadFactory threadFactory` I believe

Reply:
That will always inject a thread factory, we don't need one when the executor is provided.